pkg/auth/auth_handler.go

```diff
@@ -176,7 +180,11 @@ func (a *AuthHandler) AuthUserAuth(token string) (interface{}, error) {
 
 	err = a.storeAdminInPayload(payload)
 	if err != nil {
-		return nil, err
+		a.log.Errorf("Unable to fetch user's capabilities: %v", err)
```

Comment:
It's the same code in both places, so maybe it should move into a function or handled by `storeAdminInPayload`?

Reply:
It could help us understand from which flows it comes: User or Agent suth flow

Reply:
Then it's more relevant for `IsKnownError`

Reply:
Added `getError` for common code.